scripts/inspect_cache.py

```diff
@@ -0,0 +1,24 @@
+import os
+import base64
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir))))
+from allennlp.common.file_utils import filename_to_url
+
+CACHE_ROOT = os.path.expanduser(os.path.join('~', '.allennlp'))
+DATASET_CACHE = os.path.join(CACHE_ROOT, "datasets")
```

Comment:
I think you should import both of these constants from `file_utils`, in case we decide to change them at some point

Reply:
Done :)